frontend/src/container/ListOfDashboard/DashboardTemplates/DashboardTemplatesModal.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable sonarjs/no-duplicate-string */
```

Comment:
We can update our linting errors to suit our coding styles

Reply:
👍